modules/src/em_data/makeem.c

```diff
@@ -0,0 +1,309 @@
+/** A utility to convert the em definition table to generated code.
```

Comment:
I'm not keen on this --- it replaces the existing 77 line script with 300 lines of C, which is very much the wrong language for text processing. OTOH it does remove the ed dependency, which is great. Something like awk or Lua (there's a Lua interpreter already in the build system; see mach/mips/as/build.lua for how to call it) would probably reduce the size by 2/3.


Also, this isn't hooked into the build system, which still uses the shell script.

Reply:
I have removed this from the branch, I will try to convert it to an AWK script instead.